pypose/module/dynamics.py

```diff
@@ -0,0 +1,196 @@
+        
+        Let the input be matrix :math:`A`, :math:`B`, :math:`C`, :math:`D`, :math:`x`, :math:`u`, :math:`c_1`, :math:`c_2`. 
+        :math:`x_i` represents each individual matrix in the batch. 
+        
+        Note:
+            -:math:`x`, :math:`u` could be single input or multiple inputs
+
+            -:math:`A`, :math:`B`, :math:`C`, :math:`D` can only be two-dimensional matrices or the batch
+             In the batch case, their dimensions must be the same as those of :math:`x`, :math:`u`. 
+             :math:`A`, :math:`B`, :math:`C`, :math:`D` and :math:`x`, :math:`u` are multiplied separately for each channel.
+             
+            -For a System with p inputs, q outputs, and n state variables, :math:`A`, :math:`B`, :math:`C`, :math:`D` are n*n n*p q*n and q*p constant matrices.
+             
+            -Note that variables are entered as row vectors.
+
+        Example:
+            >>> A = torch.randn((2,3,3))
+                B = torch.randn((2,3,2))
+                C = torch.randn((2,3,3))
+                D = torch.randn((2,3,2))
+                c1 = torch.randn((2,1,3))
+                c2 = torch.randn((2,1,3))
+                x = torch.randn((2,1,3))
+                u = torch.randn((2,1,2))
+            >>> A
+            tensor([[[-2.1218, -0.0558,  1.6897],
+                     [ 0.3156, -0.7407, -0.0628],
+                     [ 0.7567,  0.6081,  0.0089]],
+                    [[-0.4036,  1.0982,  0.4694],
+                     [ 0.3743,  0.7851, -1.4842],
+                     [-1.0845, -0.8888, -0.0843]]])
+                B
+            tensor([[[-0.8426, -1.3793],
+                     [ 0.2805, -0.1517],
+                     [-0.3015,  0.0862]],
+                    [[-0.4926,  0.7835],
+                     [-0.1341,  0.1560],
+                     [-0.4062,  0.3711]]]) 
+                C
+            tensor([[[-0.6730,  0.1076,  0.7406],
+                     [-0.9979,  0.9915, -1.5760],
+                     [-0.7343,  0.5490,  0.5513]],
+                    [[-0.3741, -0.7521,  0.4004],
+                     [-1.4800,  0.0868,  0.1175],
+                     [ 0.2590,  1.4560, -0.3273]]]) 
+                D
+            tensor([[[ 1.2974, -1.0183],
+                     [-0.0850, -1.9760],
+                     [ 0.7531, -0.6172]],
+                    [[-0.3685,  0.0948],
+                     [-0.9020, -2.0040],
+                     [ 0.5833,  1.0258]]])   
+                c1
+            tensor([[[ 0.9888,  0.1157, -1.2387]],
+                    [[ 0.5319, -1.3769, -0.0701]]])   
+                c2
+            tensor([[[-0.6067, -1.8852, -0.8722]],
+                    [[-0.3974,  2.1444,  0.5077]]]) 
+                x
+            tensor([[[ 0.9576,  0.6824, -2.3676]],
+                    [[-0.5359,  1.4186,  0.9334]]])  
+                u
+            tensor([[[ 1.0024,  1.9124]],
+                    [[-0.9396,  1.1412]]])
+            >>> lti = LTI(A, B, C, D, c1, c2)
```

Comment:
This example is inconsistent with that in the test script. 
lti = pp.module.LTI(...)
lti(x,u)